index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css"
+    />
+    <title>Portfolio</title>
+</head>
+<body>
+    <section>
+        <nav id="navbar" class="navbar fixed-top navbar-expand-lg navbar-dark bg-secondary">
+          <div class="container-fluid">
+            <a class="navbar-brand" href="#">John Doe</a>
+            <button class="navbar-toggler" 
+              type="button" 
+              data-bs-toggle="collapse" 
+              data-bs-target="#navbarNavAltMarkup"
+              aria-controls="navbarNavAltMarkup"
+              aria-expanded="false"
+              aria-label="Toggle navigation"
+            >
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+              <div class="navbar-nav">
+                <a class="nav-link text-white" href="#portfolio">Portfolio</a>
+                <a class="nav-link text-white" href="#about">About</a>
+                <a class="nav-link text-white" href="#contact">Contact</a>
+                <i class="bi bi-envelope text-white"></i>
+              </div>
+            </div>
+          </div>
+        </nav>
+    </section>
```

Comment:
- [ ]  The container of this section should be a `header` tag, this will make your website more semantic 👍🏿 

- [ ] Kindly make your header background color the same color used in the Figma design 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 2-Main-Mobile (1)](https://user-images.githubusercontent.com/69638013/176621865-fc7944da-019f-4c16-a064-dcf4e588a306.png)| ![Screenshot_304](https://user-images.githubusercontent.com/69638013/176621824-45c135b1-bf25-4036-9135-b34787997e36.png)